source/fundamentals/utf8-validation.txt

```diff
@@ -0,0 +1,116 @@
+.. _nodejs-utf-8-validation:
+
+================
+UTF-8 Validation
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to enable or disable the {+driver-short+}'s
+**UTF-8** validation feature. **UTF-8** is a character encoding specification
+that ensures compatibility and consistent presentation across most operating
+systems, applications, and language character sets. By default, the driver
+checks documents for any characters that are not encoded in a valid UTF-8
+format when it transfers data between your application and MongoDB.
+
+If you enable validation, the driver substitutes invalid UTF-8 characters
+with valid UTF-8 ones when transferring data between your application and
+MongoDB. The validation adds processing overhead since it needs to verify the
+data and to make substitutions when necessary.
+
+If you disable validation, your application can avoid the processing
```

Comment:
[comment] It sounds like if you disable validation, your application will definitely avoid the processing overhead
```suggestion
If you disable validation, your application avoids the processing
```

Reply:
I try to use simple present rather than present perfect ("Use the Simplest Tense" from the style guide), but am fine with the change if it's more clear.

I wasn't able to think of a great way to keep it in simple tense and try to emphasize that it definitely happens. The closest version I could think of is "If you disable validation, your application does not [incur] the processing overhead of validation" (or some other verb that is less complex than "incur").

Reply:
I'm OK with "incur" myself. Maybe not the most ubiquitous word but I know I've used it before when it fits the situation like this.